package.json

```diff
@@ -41,22 +42,18 @@
   ],
   "license": "MIT",
   "scripts": {
-    "build": "npm run clean && npm run build-alt && npm run build-utils && npm run build-alt-browser && npm run build-alt-browser-with-addons",
-    "build-alt": "babel src/alt --out-dir lib --stage 0",
```

Comment:
hmm let me play with this because I like having this at the moment so I can just selectively build alt and not have to wait until util builds